plugin/fzf.vim

```diff
@@ -44,20 +44,31 @@ if s:is_win
       let &shellslash = shellslash
     endtry
   endfunction
-
-  function! s:fzf_shellescape(path)
-    return substitute(s:fzf_call('shellescape', a:path), '[^\\]\zs\\"$', '\\\\"', '')
-  endfunction
 else
   function! s:fzf_call(fn, ...)
     return call(a:fn, a:000)
   endfunction
-
-  function! s:fzf_shellescape(path)
-    return shellescape(a:path)
-  endfunction
 endif
 
+function! s:shellesc_sh(arg)
+  return '"'.substitute(a:arg, '"', '\\"', 'g').'"'
```

Comment:
No, wait, I guess we should probably just use builtin `shellescape` if it's okay.